commit ad3d1771126b3aabb9f7f1d62482c89424276234
Author: Takashi Iwai <tiwai@suse.de>
Date:   Tue Sep 4 17:58:41 2018 +0200

    staging: bcm2835-audio: Drop unnecessary pcm indirect setup
    
    commit 7318ec896f4856fae2bb013858e422fa078201e1 upstream.
    
    The hw_queue_size of PCM indirect helper doesn't need to be set up if
    you use the whole given buffer size.  Drop the useless
    initialization, which just confuses readers.
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Tested-by: Stefan Wahren <stefan.wahren@i2se.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
index a3605505cc20..c935c6e99633 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
+++ b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
@@ -280,7 +280,6 @@ static int snd_bcm2835_pcm_ack(struct snd_pcm_substream *substream)
 	struct bcm2835_alsa_stream *alsa_stream = runtime->private_data;
 	struct snd_pcm_indirect *pcm_indirect = &alsa_stream->pcm_indirect;
 
-	pcm_indirect->hw_queue_size = runtime->hw.buffer_bytes_max;
 	return snd_pcm_indirect_playback_transfer(substream, pcm_indirect,
 						  snd_bcm2835_pcm_transfer);
 }
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1452_ad3d1771126b3aabb9f7f1d62482c89424276234.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1452_ad3d1771126b3aabb9f7f1d62482c89424276234.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
